content/en/docs/concepts/outputs/formatting.md

```diff
@@ -31,100 +33,137 @@ append_output:
   output: A file (user=%user.name command=%proc.cmdline file=%fd.name) was read after startup
-  override: 
+  override:
     output: replace
 ```
 
-## Example Rule
+### Suggested Output Fields
+
+By default, Falco can also include "suggested" fields from plugins implementing the extraction capabilities. This is especially useful if certain plugins mark some fields as recommended for output. Those fields will appear automatically in your alerts.
+
+Below is an example configuration entry that enables suggested output fields unconditionally for any source:
+
+```yaml
+append_output:
+  - suggested_output: true  # Enable the use of extractor plugins' suggested fields for all matching sources.
+```
+
+When `suggested_output` is set to `true`, any extractor plugin that provides "suggested" fields will add them to the output in the form `plugin_field_name=$plugin.field_name`.
+
+### Command-Line Usage
+
+You can also specify this option on the command line via the `-o` flag, for example:
+
+```bash
+falco ... \
+  -o 'append_output[]={"match": {"source": "syscall"}, "extra_output": "on CPU %evt.cpu", "extra_fields": ["evt.hostname"]}'
+```
+
+## Example Rule and Various Scenarios
+
+Consider the rule:
 
 ```yaml
 - rule: Drop and execute new binary in container
   desc: SKIPPED
-  condition: 
+  condition: >
     spawned_process
     and container
-    and proc.is_exe_upper_layer=true 
+    and proc.is_exe_upper_layer=true
     and not container.image.repository in (known_drop_and_execute_containers)
   output: Executing binary not part of base image (proc_sname=%proc.sname user=%user.name process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
   priority: CRITICAL
   tags: [maturity_stable, container, process, mitre_persistence, TA0003, PCI_DSS_11.5.1]
 ```
 
-*Scenario 1*
+### Scenario 1
 
-The rule outputs include `%container.info`, but Falco is started without any command line flags:
+- The rule output includes `%container.info`.
+- Falco is started without additional command-line flags:
 
-```bash 
-sudo /usr/bin/falco  -c /etc/falco/falco.yaml -r falco_rules_test.yaml
+```bash
+sudo /usr/bin/falco -c /etc/falco/falco.yaml -r falco_rules_test.yaml
 ```
 
-In this case Falco will output `%container.id` and `%container.name` but no other container metadata will be displayed:
+Output example:
 
 ```bash
 03:00:45.104332605: Critical Executing binary not part of base image (proc_sname=bash user=root process=sleep proc_exepath=/tmp/sleep parent=bash command=sleep 10000 terminal=34816 container_id=0fdb3cd5b5fc container_name=optimistic_newton)
 ```
 
-*Scenario 2*
+By default, `%container.info` includes `%container.id` and `%container.name`, but no other container metadata is shown.
+
+### Scenario 2
 
-The rule outputs include `%container.info`, and Falco is started with the `-pc` flag:
+- The rule output includes `%container.info`.
+- Falco is started with the `-pc` flag:
```

Comment:
isn't supposed to be deprecated to use `-pc`?

Reply:
It is not deprecated yet.